scipy/stats/tests/test_distributions.py

```diff
@@ -456,6 +456,15 @@ def test_gennorm(self):
         assert_almost_equal(pdf1, 2*pdf2)
 
 
+class TestLaplaceasymmetric(object):
+    def test_laplace_asymmetric(self):
```

Comment:
This tests generate Asymmetric Laplace against Laplace.  Are there any asymmetric tests that would be useful to perform?

Reply:
None of the asymmetric Laplace distributions yields known equivalent. I'm simply copying what `gennorm` already does and believe that it's an adequate test.

Reply:
Without a test of asymmetric Laplace , one could replace `stats.laplace_asymmetric` with `stats.laplace` and all tests would pass.  `stats.laplace_asymmetric` should agree with `stats.laplace` at `kappa=1`, but there must be some value of `kappa` where the `pdf/cdf/...` can be computed and compared against some stored values.

Reply:
Done thanks.

Reply:
@essandess please let @pvanmulbregt or another reviewer decide whether to click the resolve button.  It helps us keep track of where the review is and lets us check whether the changes are sufficient.

I unresolved this because I'm not sure that the comment has been addressed.  @pvanmulbregt, has this been resolved? I don't think I can see the commit @pvanmulbregt reviewed because of the force pushes, but I think `test_asymmetric_laplace` is new, and it does test an identity of the pdf. However, @pvanmulbregt seemed to want to compare againsts reference value, e.g. computed by R or published online.